app/src/main/res/layout/activity_about.xml

```diff
@@ -32,7 +32,17 @@
 
         <include layout="@layout/appbar_empty"/>
 
-        <include layout="@layout/content_about"/>
+        <ScrollView
```

Comment:
Oh man, this is pretty complicated since I also deleted a file (`content_about.xml`).
Restoring a deleted, commit and pushed file to delete in another commit? I don't even know if this is possible in git without history corruption.

Reply:
It is possible and not so complicated. `git rebase -i HEAD~1`. Than select that you want to edit the commit. Basically, Git tells you what to do. But anyway, just leave it and next time remember that you do separate commits.